Source/Core/DolphinQt/QtUtils/ModalMessageBox.cpp

```diff
@@ -6,10 +6,10 @@
 
 #include <QApplication>
 
-ModalMessageBox::ModalMessageBox(QWidget* parent)
+ModalMessageBox::ModalMessageBox(QWidget* parent, bool app_modal)
```

Comment:
Would it make more sense to pass the Qt enum directly as an argument instead of this bool, for increased clarity?

Reply:
I considered this - and since Qt stuff is defined all over the place anyway, I don't see why not. An earlier concept set `ApplicationModal` only if message box had no parent but I decided to separate those two concepts. With the way it is now it indeed makes sense.